src/main/java/com/kuliginstepan/dadata/client/domain/organization/Organization.java

```diff
@@ -97,4 +107,12 @@
         @JsonAlias("short")
         private String shortName;
     }
+
+    @Value
+    public static class Okveds {
```

Comment:
Rename class `Okveds` from plural to singlular like `Okved`, because this class represents single Okved